resources/js/components/SignUp.js

```diff
@@ -170,7 +170,7 @@ const SignUp = (props) => {
                 id="password"
                 autoComplete="new-password"
                 InputLabelProps={{ required: false }}
-                type={props.isShownPass === false ? "password" : "text"}
+                type={!props.isShownPass ? "password" : "text"}
```

Comment:
you can create a variable for this, instead of typing this 2x in this file.

```
const type = !props.isShownPass ? "password" : "text"
...
...
...

<TextField
  ...
  type={type}
  ...
```